src/humidity.c

```diff
@@ -2,34 +2,90 @@
 #include <avr/delay.h>
 #include <hih8120.h>
 
-int measureHumidity() {
-    if ( HIH8120_OK == hih8120_initialise() )
+static TaskHandle_t _HumiditySensorTaskHandle;
+
+//private fields
+static SemaphoreHandle_t _xPrintfSemaphore;
```

Comment:
`src/humidity.c:8:26: warning: '_xPrintfSemaphore' defined but not used [-Wunused-variable]
 static SemaphoreHandle_t _xPrintfSemaphore;`

Unused?